CHANGELOG.md

```diff
@@ -2,6 +2,73 @@
 
 ## [Unreleased]
 
+## [3.0.0-rc1] - 2016-05-26
+
+## Changed
+* Upgrade `abstract-leveldown` from `0.12.0` to `5.0.0` (@vweevers)
```

Comment:
Aah nice to use _from_ and _to_. We should have this format in all our changelogs.

Reply:
Sure! I figured it was important here because we skipped a few majors :)